meetup-organizing.md

```diff
@@ -0,0 +1,38 @@
+# Organizing a meetup
+
+The following details the basic process for working with the Quorum to plan a meetup, confirm a speaker and topic, and provide any needed information or materials for the meetup event.
+
+The suggested timeframe for completing each of the preparation steps are:
+
+|   Step    |   Timeframe    |
+|  ---  |  ---  |  ---  |  ---  |
+|   Confirm details    |   1 month before    |
+|   Create event    |   3 weeks before    |
+|   Publicize event    |  2 weeks before     |
+
```

Comment:
**Minor formatting issue (blocking)**: When I look at the rendered version of this table, the formatting isn't working. I think the issue is that the bars need to be lined up. Here's the fix I tried to do, but you might need to just adjust it in the code editor:

```
|   Step                     |   Timeframe             |
|  ------------------   |  --------------------  | 
|   Confirm details    |   1 month before     |
|   Create event        |   3 weeks before      |
|   Publicize event    |  2 weeks before      |
```